base/shell/cmd/cmd.c

```diff
@@ -151,7 +151,7 @@ typedef NTSTATUS (WINAPI *NtReadVirtualMemoryProc)(HANDLE, PVOID, PVOID, ULONG,
 
 BOOL bExit = FALSE;       /* indicates EXIT was typed */
 BOOL bCanExit = TRUE;     /* indicates if this shell is exitable */
-BOOL bCtrlBreak = FALSE;  /* Ctrl-Break or Ctrl-C hit */
+volatile BOOL bCtrlBreak = FALSE;  /* Ctrl-Break or Ctrl-C hit */
```

Comment:
This shouldn't be necessary.

Reply:
It seemed to be required when I was testing. I was testing on VirtualBox at the time, so I'm not sure if there were some odd timing issues.